src/coreclr/tools/aot/ILCompiler/repro/repro.csproj

```diff
@@ -38,6 +38,7 @@
       <ReproResponseLines Include="--feature:System.Linq.Expressions.CanCompileToIL=false" />
       <ReproResponseLines Include="--feature:System.Linq.Expressions.CanEmitObjectArrayDelegate=false" />
       <ReproResponseLines Include="--feature:System.Linq.Expressions.CanCreateArbitraryDelegates=false" />
+      <ReproResponseLines Include="--generateunmanagedentrypoints:System.Private.CoreLib" />
```

Comment:
Nit: Could you please move it in from of `initassembly:System.Private.CoreLib` line so that it is in the same order as in the .targets file?

Reply:
Done.